integrationtests/tests/fixtures/mod.rs

```diff
@@ -509,6 +509,7 @@ pub fn gen_local(
                 *peer,
                 peer_params.clone(),
                 federation_name.to_string(),
+                Some(1),
```

Comment:
Nit: dont need to fix now, but this struct is getting to have a lot of parameters, difficult to determine which is which at first glance